internal/rbd/controllerserver.go

```diff
@@ -235,7 +235,7 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	}
 	defer cr.DeleteCredentials()
 
-	rbdVol, err := cs.parseVolCreateRequest(ctx, req)
+	rbdVol, isBlock, err := cs.parseVolCreateRequest(ctx, req)
```

Comment:
Maybe introduce a `rbdVol.isBlock()` function or member instead? That prevents the need for the additional boolean return values and parameters